Source/AtomicWebView/WebTexture2D.cpp

```diff
@@ -392,6 +390,14 @@ void WebTexture2D::SetSize(int width, int height)
 
 #endif
 
+#if defined(ATOMIC_OPENGL) && defined(ATOMIC_PLATFORM_WINDOWS) 
+	//Force width and height to multiples of 2 so that we don't squish pixels
+	//This doesn't seem to make the text completely sharp but it makes it usable for now.
+	width = (width | 2) + 1;
+	height = (height | 2) + 1;
+	texture_->SetFilterMode(TextureFilterMode::FILTER_NEAREST);
+#endif
```

Comment:
Hrm, this could be the UV adjustment in TB, what does it look like using the 0.5f half-pixel adjustment, without the width/height adjustment and not setting the filter mode? 

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/ThirdParty/TurboBadger/renderers/tb_renderer_batcher.cpp#L241